public/app/app.ts

```diff
@@ -127,6 +128,7 @@ export class GrafanaApp {
       setPanelDataErrorView(PanelDataErrorView);
       setLocationSrv(locationService);
       setTimeZoneResolver(() => config.bootData.user.timezone);
+      initGrafanaLive();
```

Comment:
moved this out of the `GrafanaCtrl` so it is initialized regardless of the angular status!